test/tasks_screen/tasks_screen_test.dart

```diff
@@ -22,16 +25,90 @@ void main() {
     });
   }
 
+  Future<void> longPumpUntilSettle(WidgetTester tester) async {
+    await tester.pump();
+    await tester.pump();
+    await tester.pump(const Duration(seconds: 1));
+  }
```

Comment:
Can this be converted into something more clear?  Maybe it could be a shared method where a condition is passed, and 
```
while(!condition) {
    try {
        pumpUntilSettle()
    } catch(e){
    }

}
```